PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install JumpCloud Password Manager App.md

```diff
@@ -11,68 +11,70 @@ windows
 ```
 # Set $LaunchPasswordManager to $false  ON LINE 35 if you do not wish to launch the password manger after installation
 
+# Get the current logged on User
+$loggedUser = Get-WmiObject -Class Win32_ComputerSystem | Select-Object -ExpandProperty UserName
+$loggedUser = $loggedUser -replace '.*\\'
+
+# Construct the Registry path using the user's SID
+$userSID = (New-Object System.Security.Principal.NTAccount($loggedUser)).Translate([System.Security.Principal.SecurityIdentifier]).Value
+$registryPath = "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\ProfileList\$userSID"
+
+# Get the ProfileImagePath value from the Registry
+$loggedOnUserProfileImagePath = Get-ItemPropertyValue -Path $registryPath -Name 'ProfileImagePath'
+Write-Output "loggedOnUserProfileImagePath: $loggedOnUserProfileImagePath"
+
 $installerURL = 'https://cdn.pwm.jumpcloud.com/DA/release/JumpCloud-Password-Manager-latest.exe'
+if (Test-Path "$loggedOnUserProfileImagePath\AppData\Local\Temp" ) {
 
-$installerTempLocation = 'C:\Windows\Temp\JumpCloud-Password-Manager-latest.exe'
+    $installerTempLocation = "$loggedOnUserProfileImagePath\AppData\Local\Temp\JumpCloud-Password-Manager-latest.exe"
+    Write-Output "installerTempLocation: $installerTempLocation"
```

Comment:
Again, let's make this human readable:

"Installer Location" vs "installerTempLocation"